hawkular-alerts-api/src/main/java/org/hawkular/alerts/api/model/event/Alert.java

```diff
@@ -181,6 +184,31 @@ public void addNote(String user, String text) {
         getNotes().add(new Note(user, text));
     }
 
+    public List<LifeCycle> getLifecycle() {
+        if (null == lifecycle) {
+            this.lifecycle = new ArrayList<>();
```

Comment:
Should this initially be OPEN?  Or should LifeCycle really be more like StatusChange?